scripts/autotools-merge-lipo.cmake.in

```diff
@@ -0,0 +1,285 @@
+# Copyright (c) 2015 Ruslan Baratov, Alexandre Pretyman
+# All rights reserved.
+
+### Input params check
+
+string(COMPARE EQUAL "@multi_arch_install_root@" "" is_empty)
+if(is_empty)
+  message(FATAL_ERROR "multi_arch_install_root is empty")
+endif()
+
+string(COMPARE EQUAL "@ios_architectures@" "" is_empty)
+if(is_empty)
+  message(FATAL_ERROR "ios_architectures is empty")
+endif()
+
+string(COMPARE EQUAL "@HUNTER_PACKAGE_INSTALL_PREFIX@" "" is_empty)
+if(is_empty)
+  message(FATAL_ERROR "HUNTER_PACKAGE_INSTALL_PREFIX is empty")
+endif()
+
+set(ios_architectures @ios_architectures@)
+set(built_arch_roots)
+foreach(x ${ios_architectures})
+  list(APPEND built_arch_roots @multi_arch_install_root@/${x})
+endforeach()
+list(LENGTH ios_architectures total_arch_number)
+
+
+# We work with the first root path: we move everything, except the *.a static
+# library files into @HUNTER_PACKAGE_INSTALL_PREFIX@, then we lipo the static
+# libraries together into @HUNTER_PACKAGE_INSTALL_PREFIX@
+list(GET built_arch_roots 0 first_built_root)
+file(GLOB_RECURSE
+    library_files
+    RELATIVE
+      "${first_built_root}"
+    ${first_built_root}/*.a
+)
+
+file(GLOB_RECURSE
+    libtool_la_files
+    RELATIVE
+      "${first_built_root}"
+    ${first_built_root}/lib/*.la
+)
+
+# Remove all libtool_la_files from all built roots
+foreach(arch @ios_architectures@)
+  foreach(file_name ${libtool_la_files})
+    file(REMOVE
+        "@multi_arch_install_root@/${arch}/${file_name}"
+    )
+  endforeach()
+endforeach()
+
+file(GLOB_RECURSE
+    all_except_library_files
+    RELATIVE
+      "${first_built_root}"
+    ${first_built_root}/*
+)
+# Remove the library files
+foreach(x ${library_files})
+  list(REMOVE_ITEM all_except_library_files ${x})
+endforeach()
+
+
+# The preprocessor macros below are from
+# http://sourceforge.net/p/predef/wiki/Architectures/
+# except armv7, which was taken from:
+# clang -arch armv7 -dD -E config.h
```

Comment:
Cool, was not thinking about '-arch'.

Improved script: https://github.com/ruslo/hunter/commit/9e0494913b652ef66b842e3e0609c48d2ba6cb3a

Updated:
 * https://github.com/ruslo/hunter/commit/3a47834642ac5f3c9765a3982ee955c0ba8e48ef
 * https://github.com/ruslo/hunter/commit/83572273937bf759449cfa8430358503a6bbf7d0
 * https://github.com/ruslo/hunter/commit/f4283306fe15d82297fd28f3a101c19f61847046
 * https://github.com/ruslo/hunter/commit/24aab1606b76b4e57e68b2e008aa098dfe8d14ce